libs/community/langchain_community/callbacks/openai_info.py

```diff
@@ -41,6 +41,7 @@
     "gpt-4o-2024-08-06": 0.0025,
     "gpt-4o-2024-08-06-cached": 0.00125,
     "gpt-4o-2024-11-20": 0.0025,
+    "gpt-4o-2024-11-20-cached": 0.0025,
```

Comment:
Is this correct? Are cached tokens not 0.00125, same as `gpt-4o-08-06`?

https://openai.com/api/pricing/

Reply:
@ccurme Yes, I was wrong. I'm sorry. I fixed it.